tests/navjoy_translator_test.py

```diff
@@ -1061,3 +1075,614 @@ def test_get_types_of_work_empty():
+                    "coordinates": [
+                        [
+                            [
+                                -103.17130040113868,
+                                40.625392709715676
+                            ],
+                            [
+                                -103.17889641706886,
+                                40.61979008921054
+                            ]
+                        ]
+                    ],
+                },
+            ],
+            "streetNameFrom": "US-34",
+            "direction": "eastbound",
+            "requestedTemporarySpeed": "45",
+            "workStartDate": "2021-08-09T13:00:00.000Z",
+            "workEndDate": "2021-09-24T23:00:00.000Z",
+            "reductionJustification": "Crews roadside.",
+            'currentPostedSpeed': None,
+            'mileMarkerEnd': None,
+            'mileMarkerStart': None,
+
+            "streetNameFrom2": "US-34",
+            "directionOfTraffic2": " East/West ",
+            "requestedTemporarySpeed2": "45",
+            "workStartDate2": "2021-08-09T13:00:00.000Z",
+            "workEndDate2": "2021-09-24T23:00:00.000Z",
+            'currentPostedSpeed2': None,
+            'mileMarkerEnd2': None,
+            'mileMarkerStart2': None,
+        }
+    },
+        {
+        "sys_gUid": "Form568-cb0fdaf0-c27a-4bef-aabd-442615dfb2d6",
+        "data": {
+            "srzmap": [
+                {
+                    "type": "LineString",
+                    "coordinates": [
+                        [
+                            [
+                                -103.17130040113868,
+                                40.625392709715676
+                            ],
+                            [
+                                -103.17889641706886,
+                                40.61979008921054
+                            ]
+                        ]
+                    ],
+                },
+            ],
+            "streetNameFrom": "US-34",
+            "direction": "westbound",
+            "requestedTemporarySpeed": "45",
+            "workStartDate": "2021-08-09T13:00:00.000Z",
+            "workEndDate": "2021-09-24T23:00:00.000Z",
+            "reductionJustification": "Crews roadside.",
+            'currentPostedSpeed': None,
+            'mileMarkerEnd': None,
+            'mileMarkerStart': None,
+
+            "streetNameFrom2": "US-34",
+            "directionOfTraffic2": " East/West ",
+            "requestedTemporarySpeed2": "45",
+            "workStartDate2": "2021-08-09T13:00:00.000Z",
+            "workEndDate2": "2021-09-24T23:00:00.000Z",
+            'currentPostedSpeed2': None,
+            'mileMarkerEnd2': None,
+            'mileMarkerStart2': None,
+        }
+    }]
+    actual = navjoy_translator.expand_speed_zone(event)
+
+    assert expected == actual
+
+
+# I hate how long this test is, but this is what is has to be to test all 4 at the same time
```

Comment:
Not sure it has to be done, but what do you think about extracting these "expected" values to JSON files in a separate directory (maybe just `./ExpectedTestResults`)? It would clean up the tests a bit, we'd just have to read/parse the docs.

Reply:
Broke these out into another file, called navjoy_translator_test_expected_results.py